service/service.py

```diff
@@ -32,28 +44,14 @@
 # Import Flask application
 from . import app
 
-######################################################################
-# Configure Swagger before initilaizing it
-######################################################################
-api = Api(app,
-          version='1.0.0',
-          title='Wishlist REST API Service',
-          description='This is a sample Wishlist server for an e-commerce website.',
-          default='Wishlists',
-          default_label='Wishlist operations',
-          doc='/apidocs/index.html'
-         )
-
-# Define the model so that the docs reflect what can be sent
-wishlist_model = api.model('Wishlist', {
-    'id': fields.Integer(readOnly=True,
-                         required=False,
-                         description='The unique id assigned internally by service'),
-    'name': fields.String(required=True,
-                          description='The name of the Wishlist'),
-    'customer_id': fields.Integer(required=True,
-                                  description='The id of the customer that owns the wishlist')
-})
+#Autorizations for Swagger docs
```

Comment:
we don't need authorizations for our app

Reply:
Removed